nautobot/core/settings.py

```diff
@@ -569,6 +571,11 @@
 }
 
 CONSTANCE_CONFIG = {
+    "ALLOW_REQUEST_PROFILING": ConstanceConfigItem(
```

Comment:
For completeness/consistency, we should be able to fallback to a `settings` variable by the same name, e.g. as a commented-out item in `nautobot/core/templates/nautobot_config.py.j2` similar to how the other Constance-capable settings are handled presently.